workbench_access.post_update.php

```diff
@@ -42,20 +42,33 @@ function workbench_access_post_update_convert_to_scheme() {
       'bundles' => array_keys($config->get('fields')['node']),
     ];
   }
-  $scheme = AccessScheme::create([
-    'id' => 'default',
-    'label' => $config->get('label'),
-    'plural_label' => $config->get('plural_label'),
-    'scheme' => $config->get('scheme'),
-    'scheme_settings' => $settings,
-  ]);
-  $scheme->save();
+  // Let other modules intervene for additional types.
+  \Drupal::moduleHandler()->alter('workbench_access_scheme_update', $settings);
+
+  // No settings? Do nothing but mark the update.
+  if (empty($settings)) {
+    $message = t('Workbench Access has not been configured. Disabling the module is recommended.');
+  }
+  else {
+    $scheme = AccessScheme::create([
+      'id' => 'default',
+      'label' => $config->get('label'),
+      'plural_label' => $config->get('plural_label'),
+      'scheme' => $config->get('scheme'),
+      'scheme_settings' => $settings,
+    ]);
+    $scheme->save();
+  }
+
   \Drupal::state()->set('workbench_access_upgraded_scheme_id', 'default');
```

Comment:
This should go inside the else above it, otherwise the legacy views stuff will kick in and try to load a non-existent scheme

Reply:
I accounted for that in the change to workbench_access_views_data_alter() above, which only fires if the default scheme is set.

I think this comment might be on an outdated commit.

Reply:
Yes, I moved it in a later commit.